src/components/ng-redux.spec.ts

```diff
@@ -251,8 +251,53 @@ describe('NgRedux Observable Store', () => {
 
     ngRedux.configureStore(rootReducer, defaultState);
   });
+
+  describe('Lazy Loaded Reducers', () => {
+
+    describe('when creating the store', () => {
+      it('should throw if reducers are not provided inside constructor', () => {
+        ngRedux = new NgRedux<IAppState>(mockNgZone);
+        expect(ngRedux.configureStore.bind()).toThrowError();
+      });
+
+      it('should get the initial state when reducers have been provided through forRoot', done => {
+        ngRedux = new NgRedux(mockNgZone, { root: rootReducer });
+        ngRedux.configureStore();
+        ngRedux.select(state => state)
+          .subscribe(state => {
+            expect(state.root.foo).toEqual('bar');
+            expect(state.root.baz).toEqual(-1);
+            done();
+          });
+      });
+    });
+
+    describe('when adding reducers', () => {
+      const addedReducer = { child: (state = 'lazy', action) => state };
+
+      it('should throw if initial reducers not provided inside constructor', () => {
+        ngRedux = new NgRedux<IAppState>(mockNgZone);
+        ngRedux.configureStore(rootReducer, defaultState);
+        expect(ngRedux.addReducers.bind(addedReducer)).toThrowError();
+      });
+
+      it('should successfully add the new reducers to the state', done => {
+        ngRedux = new NgRedux(mockNgZone, { root: rootReducer });
+        ngRedux.configureStore();
+        ngRedux.addReducers(addedReducer);
+        ngRedux.select(state => state)
+          .subscribe(state => {
+            expect(state.root.foo).toEqual('bar');
+            expect(state.root.baz).toEqual(-1);
+            expect(state.child).toEqual('lazy');
+            done();
+          });
+      });
```

Comment:
what would you think about an small test case just to make sure if you add a reducer with an already existing key, that would override the existing reducer?